building/build-debs/homeworld-admin-tools/src/authority.py

```diff
@@ -15,7 +15,9 @@ def get_targz_path(check_exists=True):
         command.fail("authorities.tgz does not exist (run spire authority gen?)")
     return authorities
 
-
+def encrypted_name(filename):
+    return filename + ".encrypted"
+    
```

Comment:
It might be a good idea to have a slightly more descriptive name: this makes it sound like the name itself is getting encrypted. Maybe `name_for_encrypted_file`?